src/js/components/RangeInput/RangeInput.js

```diff
@@ -1,22 +1,34 @@
-import React from 'react';
-import { compose } from 'recompose';
-
-import { withFocus, withForwardRef } from '../hocs';
+import React, { forwardRef, useState } from 'react';
 
 import { StyledRangeInput } from './StyledRangeInput';
 
-const RangeInput = ({ forwardRef, ...rest }) => (
-  <StyledRangeInput {...rest} ref={forwardRef} type="range" />
-);
+const RangeInput = forwardRef(({ onFocus, onBlur, ...rest }, ref) => {
+  const [focus, setFocus] = useState();
+  return (
+    <StyledRangeInput
+      ref={ref}
+      focus={focus}
+      {...rest}
```

Comment:
Semantically, I think that `...rest` should be placed as the last prop so the user can override any default behavior our component provides out of the box.
In this case, I noticed you moved the `...rest` to be the third prop rather than the first, but I'm curious why it is not the latter?

Reply:
The two props after `...rest` are `onFocus` and `onBlur`, which we cannot let the caller replace. I know they won't be in `rest`, because they are excluded above, but this approach seemed to make it even clearer.